ksql-engine/src/main/java/io/confluent/ksql/metrics/ConsumerCollector.java

```diff
@@ -139,12 +139,30 @@ public void close() {
     topicSensors.values().forEach(v -> v.close(metrics));
   }
 
+  @Override
   public Collection<TopicSensors.Stat> stats(String topic) {
     final List<TopicSensors.Stat> list = new ArrayList<>();
     topicSensors.values().stream().filter(counter -> counter.isTopic(topic)).forEach(record -> list.addAll(record.stats()));
     return list;
   }
 
+  @Override
+  public double currentMessageProductionRate() {
```

Comment:
this can be removed if you stub in the Interface using

  default double currentMessageConsumptionRate() {
        return 0;
  }